index.html

```diff
@@ -6,46 +6,150 @@
+    <div id="textcontainerhome1">
+    <h3 id="spacingtop">"AI, friend or foe?"</h3>
+    <h1 id="h1desk">ARTIFICIAL INTELLIGENCE GLOBAL SUMMIT 2023</h1>
+    <div class="textbox">How should we plan ahead for the disruption that AI will bring to our world? Delegates from over 20 countries will be sharing their expertise with us. We look forward to hosting you in Cape Town, South Africa for this unique event.</div>
+    <h2 class="contdesk">2023.10.15(THU) ~ 16(FRI)</h2>
+    <h5 class="contdesk">@ Cape Town Convension Centre (897 JC Le Roux street)." </h5>
+ </div>
  </div>
-    
 </section>
+
 <section id="home2">
-    <div class="container cont2" ></div>
+    <div class="bg-image"></div>
+    <div class="container cont2" >
+    <h2 id="h2home2">Main Program</h2>
+    <div class="line"></div>
+
+    <div class="program-container">
+
+    <div class="program program1">
+        <img id="programicons" src="./Assets/Icons/1.lecture.svg" alt="lecture">
+        <h4>Lecture</h4>
+        <p id="programtext">Hot topics in Artificial Inteligence from our global and local speakers.</p>
+    </div>
+    <div class="program program2">
+        <img id="programicons" src="./Assets/Icons/2.ccexhibition.svg" alt="exhibition">
+        <h4>Exhibition</h4>
+        <p id="programtext">The best exibits from local schools and universities will be available for viewing. </p>
+    </div>
+    <div class="program program3">
+        <img id="programicons" src="./Assets/Icons/3.forum.svg" alt="exhibition">
+        <h4>Forum</h4>
+        <p id="programtext">Most sessions will be accompanied by an forum where questions can be posed to the speakers. </p>
+    </div>
+    <div class="program program4">
+        <img id="programicons" src="./Assets/Icons/4.workshop.svg" alt=workshop">
+        <h4>Workshop</h4>
+        <p id="programtext">Various workshops will be held, during the plenary sessions.</p>
+    </div>
+    <div class="program program5">
+        <img id="programicons" src="./Assets/Icons/5.CC Ignite.svg" alt=workshop">
+        <h4>Network</h4>
+        <p id="programtext">An opportunity to network with AI leaders accross the globe.</p>
+    </div>
+    </div>
+
+   <button class="button hidden" type="button">Join CC Global Summit 2025</button>
+   <p id="seeprogram">SEE THE WHOLE PROGRAM</p>
+
+    </div>
 </section>
+
 <section id="home3">
-    <div class="container cont3"></div>
+    <div class="container cont3">
+    <h2 id="h2home3">Featured Speakers</h2>
+    <div class="line"></div>
+    <div class="cards"></div>
+
+    <div class="more hidden">
+            <button id="more" class="button more" type="button">MORE<i class="fas fa-chevron-down"></i></button>
+    </div>
+
+    </div>
 </section>
 
+<section id="home4">
+    <div class="container cont7 cont9" >
+    <h2 id="h2about4">Partner</h2>
+    <div id="linehome4" class="line"></div>
+    <div class="partner">
+        <ul class="brands">
```

Comment:
- [ ] When the screen size is at 100%, your brand logos overlap their container. Please position these as seen in the original design.
### Your Implementation
![image](https://github.com/dvcorreia1984/Capstone-Project/assets/45460192/62d5f9df-b49d-4d2e-a2ca-8f3dece7edd7)
### Required
![image](https://github.com/dvcorreia1984/Capstone-Project/assets/45460192/eb841d21-65c6-4f82-8fde-afd1c4cfd72d)

Reply:
Ok, I improved my version.